ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/connected-endpoints-monitor-session-mismatch-acl.yml

```diff
@@ -0,0 +1,32 @@
+loopback_ipv4_pool: 192.168.1.0/24
+
+type: l2leaf
+l2leaf:
+  defaults:
+  nodes:
+    - name: connected-endpoints-monitor-session-mismatch-acl
+
+servers:
+  - name: INDIVIDUAL_1
+    adapters:
+      - switches: [connected-endpoints-monitor-session-mismatch-acl]
+        switch_ports: [Ethernet14]
+        description: Monitor port-channel 44
+        port_channel:
+          channel_id: 14
+          mode: active
+        monitor_sessions:
+          - name: DMF
+            role: source
+            source_settings:
+              access_group:
+                type: ip
+                name: acl2
+            session_settings:
+              access_group:
+                type: ip
+                name: acl1
+
+expected_error_message: >-
+  For connected-endpoints-monitor-session-mismatch-acl,
+  an ACL is set on Port-Channel14 even though ACL is set for monitor session DMF in connected endpoints settings.
```

Comment:
```suggestion
  Can not set ACL for connected-endpoints-monitor-session-mismatch-acl Port-Channel14 since it is already present on monitor session DMF in connected endpoints settings.
```